build.gradle

```
@@ -306,30 +310,37 @@ android {
       simprintsModuleId = 'simprints.app'
       simprintsUserId = 'test@simprints.app'
     }
+
     vhw_burundi {
       dimension = 'brand'
       applicationId = 'org.medicmobile.webapp.mobile.vhw_burundi'
     }
-    surveillance_covid19_kenya {
+
+    urveillance_covid19_kenya {
```

Looks like you've accidentally deleted too much here...

```suggestion
    surveillance_covid19_kenya {
```

---

[Face palm] Thanks for the catch! It's fixed